db/init_db.js

```diff
@@ -30,9 +32,11 @@ async function buildTables() {
           name VARCHAR(255) NOT NULL, 
           description VARCHAR(1000), 
           price INTEGER NOT NULL, 
+          stripe_price_id VARCHAR(255),
```

Comment:
it looks like we accidentally added this twice

Reply:
removed